modular_skyrat/modules/food_replicator/replicator.dm

```diff
@@ -0,0 +1,23 @@
+#define RND_CATEGORY_NRI_FOOD "Provision"
+#define RND_CATEGORY_NRI_MEDICAL "Medicine"
+#define RND_CATEGORY_NRI_CLOTHING "Apparel"
+
+/obj/machinery/biogenerator/food_replicator
+	name = "\improper Type 34 'Colonial Supply Core'"
```

Comment:
It's not called Food Replicator here, but you refer to it as the food replicator elsewhere. You should probably settle on one thing and stick with it.

Reply:
Will rename the board too next commit.

Reply:
Having the path and the item name be the same helps for people trying to find it in code, or spawning it, and consistency is generally preferred. Seeing it listed on cargo also as "food replicator" also gives it the clarity it should have. The fancy name can be in the description